features/preferences/impl/src/main/kotlin/io/element/android/features/preferences/impl/advanced/AdvancedSettingsPresenter.kt

```diff
@@ -78,6 +147,9 @@ class AdvancedSettingsPresenter @Inject constructor(
             isSharePresenceEnabled = isSharePresenceEnabled,
             theme = theme,
             showChangeThemeDialog = showChangeThemeDialog,
+            pushDistributor = currentDistributorName,
```

Comment:
rename `currentPushDistributor` to be clear?

Reply:
4d6cdf03412270f31bffb0fe963d5c8695e2fe53